pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2049,6 +2197,56 @@ func xlShutdown(domainName string, domainId int, force bool) error {
 	return nil
 }
 
+// Wrapper for domain Destroy thru xlDestroy or rktRm
+func wrapDomainDestroy(status types.DomainStatus) error {
```

Comment:
wouldn't a more natural name for this function simply be domainDestroy? I mean containers ARE domains in our implementation.

Reply:
Taken care of in latest commit